ios/NotifeeCore/NotifeeCore+UNUserNotificationCenter.m

```diff
@@ -133,10 +133,22 @@ - (void)userNotificationCenter:(UNUserNotificationCenter *)center
 - (void)userNotificationCenter:(UNUserNotificationCenter *)center
     didReceiveNotificationResponse:(UNNotificationResponse *)response
              withCompletionHandler:(void (^)(void))completionHandler {
-  NSDictionary *notifeeNotification =
-      response.notification.request.content.userInfo[kNotifeeUserInfoNotification];
+  NSMutableDictionary *notifeeNotification =
+      [response.notification.request.content.userInfo[kNotifeeUserInfoNotification] mutableCopy];
+
+  NSLog(@"notifee didReceiveNotificationResponse notifee %@", notifeeNotification);
+
+  // check if we should handle notification created outside of notifee
+  if (notifeeNotification == nil) {
+    NSDictionary *userInfo = response.notification.request.content.userInfo;
+    BOOL shouldHandleNotification = [userInfo[kNotifeeUserInfoNotifee] boolValue];
```

Comment:
I don't suppose there is a way to test these? It appears possible:

> $ xcrun simctl push <device> com.example.my-app ExamplePush.apns"

https://betterprogramming.pub/how-to-send-push-notifications-to-the-ios-simulator-2988092ba931

I'm always concerned about null handling so seeing a test of all branches would be nice (notification has no userInfo? Is that possible? notification has key, notification doesn't have key...)

Reply:
yeah good point. I've removed this bool check... so we now just default in handling remote notifications on iOS